2001-3000/LIT2361somade.xml

```diff
@@ -27,6 +27,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <p>Work of the literatures of Ethiopia and Eritrea</p>
             <p>A TEI file converted from the ETHIO-authority google spreadsheet</p>
          </sourceDesc>
+            
+               <sourceDesc>
+                  <listWit>
+                     <witness  corresp="BAVet87">V1</witness>             
```

Comment:
as mentioned, you do not need to provide any text inside of the witness node unless you need to add something (foliation, date, etc) that you want to have printed - in this case the shelfmark must be provided according to the standard
(e.g. Paris, Bibliothèque nationale de France, Éthiopien 87) etc.

If you want to specify the sigla used by Velat in the edition you can at least then write out what you want to see printed in `witness`
(
```
<witness  corresp="BAVet87">Paris, Bibliothèque nationale de France, Éthiopien 87, sigla V1 in  <bibl>
                     <ptr target="bm:Velat1966Soma"/>
                     <citedRange unit="page">254–261</citedRange>
                  </bibl></witness>
```

etc)

but actually, manuscript sigla belong to manuscript descriptions and can be mentioned there in the bibliography or as idno

Reply:
Ok, thank you! I will remove it!